packages/core/saved-objects/core-saved-objects-migration-server-internal/src/model/model.ts

```diff
@@ -1522,6 +1522,12 @@ export const model = (currentState: State, resW: ResponseType<AllActionStates>):
             },
           ],
         };
+      } else if (isTypeof(left, 'types_added')) {
+        // compatible migration: ONLY new SO types have been introduced, skip directly to UPDATE_TARGET_MAPPINGS_META
+        return {
+          ...stateP,
+          controlState: 'UPDATE_TARGET_MAPPINGS_META',
```

Comment:
your test makes it look like this is working, but I'm unsure when do we actually update the mappings `properties` since UPDATE_TARGET_MAPPINGS_META specifically doesn't update the properties https://github.com/elastic/kibana/blob/main/packages/core/saved-objects/core-saved-objects-migration-server-internal/src/next.ts#L240

Reply:
The mappings `properties` are already updated in the `UPDATE_SOURCE_MAPPING_PROPERTIES` step.

Reply:
For your peace of mind, I did test this with the [integration test below](https://github.com/elastic/kibana/pull/197061/files#diff-6247cbdf88b802cb60c2a76176f2fec677d004d62cd0ab0c964e8258239881b3):

```typescript
expect(mappings.properties?.recent).toEqual({
  properties: {
    name: {
      type: 'keyword',
    },
  },
});
```

Reply:
Yeah I saw the test, so knew I must be missing something, thanks that makes sense!